xee/ext.py

```diff
@@ -689,7 +701,7 @@ def _parse_dtype(data_type: types.DataType):
 
 
 def _ee_bounds_to_bounds(bounds: ee.Bounds) -> types.Bounds:
-  coords = np.array(bounds['coordinates'], dtype=np.float32)[0]
+  coords = np.array(bounds['coordinates'], dtype=np.float64)[0]
```

Comment:
I want to sanity check what I’m reading. This is just for coords/dimensions, right? (I’m on mobile 😉).

Reply:
If so, I see how this change is crucial.

Reply:
Correct, this is a private method used only to build coords.